### PR TITLE
Add accessibility guidelines

### LINKED ISSUE #1718: [Enhancement] Guidelines for Accessibility and voice over
## Describe the enhancement
As a developer, I wish to be able to navigate to cookbook.kirby and get guidelines to build my html accessible and voiceover enabled. 

## Describe the solution you'd like
A collection of guidelines to correctly build html pages regarding to accessibility and with voice over focus. 

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](../CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](../CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](../CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by MadsBuchmann**
## Which issue does this PR close?

This PR closes #1718

## What is the new behavior?

Accessibility guidelines has been added. While this PR is in draft they are stored at top level in the repo in the file `accessibility-guidelines.md`. This is done for the sake of gathering feedback. 

Once the actual content of the guidelines are settled I will find a proper place for them in the cookbook and convert it to HTML 😄

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] ~~Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".~~
- [ ] ~~Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).~~

### Review  
- [X] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be automatically merged to master via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).

### CODE REVIEW

File: accessibility-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Accessibility in Kirby
+
+[ Index goes here when converted to HTML]
+
+## Can Kirby make an application accessible?
+
+The short answer is no.
+
+The accessibility of an application largely depends on its markup. In other words, it depends on how the components in Kirby is used. The role of the components is to allow for developers to make their applications accessible; they should not be in the way.
```
Comment (jkaltoft) [inline accessibility-guidelines.md:9]:
```suggestion
The accessibility of an application largely depends on its markup. In other words, it depends on how the components in Kirby are used. The role of the components is to allow for developers to make their applications accessible; they should not be in the way.
```

File: accessibility-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+In an attempt to make it easier to get started with accessibility, the following section details our mindset towards accessibilty and how we think about it in Team Kirby.
+
+#### Accessibility is not a feature
+
+Accessibility is not something that is tagged onto an application like a feature.
+
+**All** applications have some degree of accessibility, whether it is considered or not. If it is not actively prioritized, the accessibility of an application is most likely poor. It is similar to the quality of a codebase; simply by existing it has some degree of quality. Not prioritizing it might have consequences in the long run, such as the codebase being hard to modify. Analogously not prioritizing accessibility might make an application impossible to use for people with various disabilities.
+
+Then how can it be improved? Beginning applying ARIA tags and using automated tests such as lighthouse, might spring to mind. This is definetly better than nothing. However accessibility is more than that.
```
Comment (jkaltoft) [inline accessibility-guidelines.md:25]:
```suggestion
Then how can it be improved? Beginning applying ARIA tags and using automated tests such as Lighthouse, might spring to mind. This is definitely better than nothing. However accessibility is more than that.
```

**Review by jkaltoft — CHANGES_REQUESTED** [review]
I've added a few comments, mostly regarding spelling mistakes. 

I suggest adding more emphasis on the importance of semantic markup when we're talking web accessibility (which we are in this case).

File: accessibility-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+And it might seem overwhelming... So besides helping those with disabilities to access and use the digital tools we all rely on in the 21st century; why care as a developer without any disabilities?
+
+So here is a more ego-centric argument for caring about accessibility: you will most likely be disabled to some degree in the future. Apologies if it sounds bleak.
+
+You might break your dominant arm causing you to rely on keyboard navigation. More severely you could get into a car accident paralyzing you from the neck down. Moreover your sight, motorskills and reaction time will most likely decrease as you get older.
+
+In any case would you be able to rely on the digital tools you do now in a world where developers does not consider accessibility?
+
+## Kirby's role
```
Comment (jkaltoft) [inline accessibility-guidelines.md:55]: Consider a comparison of the role of Kirby components to HTML elements; they can be used both in a few "correct" ways and also in a lot of completely inaccessible ways

**Review by jkaltoft — CHANGES_REQUESTED** [review]
I found a few more spelling mistakes.

In general, I think these are good guidelines.

In addition I suggest putting the following points even more front and center, perhaps as a TL;DR bulleted list:

* Accessibility is hard - but it's even harder (and more expensive) - to improve accessibility retrospectively
* Make accessibility your top priority as a frontend web developer - because it _is_ your job
    - https://gomakethings.com/accessibility-is-hard.-its-also-your-job./
    - https://www.linkedin.com/pulse/so-you-want-learn-front-end-web-development-heres-how-morten/
* Learn to review accessibility in your application - hint: You can't rely on automatic testing alone
* Write proper semantic standards based markup and apply progressive enhancement principles
    - This is how you gain the most with the least effort
    - Proper markup not only benefits people with disabilities - [accessibility is a win for everybody](https://uxdesign.cc/the-unseen-benefits-of-accessibility-ef259202b638)
    - Proper markup is also easier for machines/computers to process, which makes your application more robust, more future proof and more awesome
* Learn how, when and when _not_ to use ARIA attributes

More resources:

* [Simplifying web development with accessibility](https://www.linkedin.com/learning/simplifying-web-development-with-accessibility-best-practices/simplifying-web-development-with-accessibility)
* [Accessibility for Web Design](https://www.linkedin.com/learning/accessibility-for-web-design/?lipi=urn%3Ali%3Apage%3Ad_flagship3_pulse_read%3BoXC9j85USFiTVtZZ72hM7w%3D%3D)

File: accessibility-guidelines.md (hunk 3 above)
Comment (MadsBuchmann) [reply accessibility-guidelines.md:55]: I've attempted to add it here: https://github.com/kirbydesign/designsystem/pull/1851/commits/682a4a07ee5719096762df9f4b4fa7fcabbe7241

**Comment by MadsBuchmann** [comment]
@jkaltoft ready for next round! I've attempted to include all of your points without ending up making the guidelines too long 😄

**Review by jkaltoft — CHANGES_REQUESTED** [review]
Nice work.

I found a few more spelling mistakes.

I suggest promoting the W3C WCAG specification and WebAIM resources. I think they are more than "additional resources".

I also think we should consider if there's somewhere we can mention the WAVE browser extension. It's a very good tool.

Other than that, I think we're good 🙂

**Review by jkaltoft — APPROVED** [review]
Good work!

If you want to do more, consider revisiting heading levels. Not a requirement at this point 🙂

**Pull request merged**